docs/index.rst

```diff
@@ -31,14 +31,14 @@ standard libraries:
 
 urllib3 is powerful and easy to use:
 
-.. code-block:: pycon
-
-   >>> import urllib3
-   >>> resp = urllib3.request("GET", "https://httpbin.org/robots.txt")
-   >>> resp.status
-   200
-   >>> resp.data
-   "User-agent: *\nDisallow: /deny\n"
+.. code-block:: python
```

Comment:
Similarly to `README` we'll keep this one.

Reply:
Similar to `README` brought back the `pycon` block but removed the `>>>` and retained the print comments.